Realm/RLMObjectStore.mm

```diff
@@ -90,17 +90,24 @@ void RLMInitializeSwiftAccessorGenerics(__unsafe_unretained RLMObjectBase *const
     }
 
     for (RLMProperty *prop in object->_objectSchema.swiftGenericProperties) {
+        id ivar = object_getIvar(object, prop.swiftIvar);
+        if (!ivar) {
+            // FIXME: this should actually be an error as it's the result of an
```

Comment:
Is there an issue tracking this, so we remember fixing it for the next major version?